vscode-dotnet-runtime-library/src/Utils/WebRequestWorker.ts

```diff
@@ -112,8 +155,35 @@ export class WebRequestWorker
             throw formattedError;
         }, this.websiteTimeoutMs);
 
+        // Make the web request
         const response = await this.client.get(url, { signal: timeoutCancelTokenHook.signal, ...options });
+        // Timeout for Web Request -> Not Timer. (Don't want to introduce more CPU time into timer)
         clearTimeout(timeout);
+
+        // CDN Timer
+        // Standard timeout time in NS : 60,000,000,000 is < than std max_safe_int_size: 9,007,199,254,740,991
+        const timerPrecision = 2; // decimal places for timer result
```

Comment:
Two decimal places for the time in nanoseconds? That's very precise

Reply:
millisecond level precision is good enough.

Reply:
The final precision is for ms, so I think this should be good.